src/CourseAuthoringPage.jsx

```diff
@@ -19,17 +23,28 @@ export default function CourseAuthoringPage({ courseId, children }) {
   const courseNumber = courseDetail ? courseDetail.number : null;
   const courseOrg = courseDetail ? courseDetail.org : null;
   const courseTitle = courseDetail ? courseDetail.name : courseId;
+  const courseAppsApiStatus = useSelector(getCourseAppsApiStatus);
+  const inProgress = useSelector(getLoadingStatus) === RequestStatus.IN_PROGRESS;
+  if (courseAppsApiStatus.status === RequestStatus.DENIED) {
```

Comment:
if we dont' store status object - the selector can simply return the status code. 
`courseAppsApiStatus === RequestStatus.DENIED`